driver/lib/sessions/session.ts

```diff
@@ -1,8 +1,27 @@
 import { FlutterDriver } from '../driver';
 import { log } from '../logger';
 
-import { DRIVER_NAME as ANDROID_DEVICE_NAME, startAndroidSession } from './android';
-import { DRIVER_NAME as IOS_DEVICE_NAME, startIOSSession } from './ios';
+import { DRIVER_NAME as ANDROID_DEVICE_NAME, startAndroidSession, connectAndroidSession } from './android';
+import { DRIVER_NAME as IOS_DEVICE_NAME, startIOSSession, connectIOSSession } from './ios';
+
+export const reConnectFlutterDriver = async function(this: FlutterDriver, caps: any) {
+  // setup proxies - if platformName is not empty, make it less case sensitive
+  if (caps.platformName !== null) {
+    const appPlatform = caps.platformName.toLowerCase();
+    switch (appPlatform) {
+      case `ios`:
+        await connectIOSSession(this.proxydriver, caps)
+        break;
+      case `android`:
+        await connectAndroidSession(this.proxydriver, caps)
```

Comment:
Re-connected session got below error by the Flutter.
It occurred even when the reconnected one started the activity with the same command as the new session request.


```
[HTTP] --> GET /wd/hub/session/3b2c33c6-c325-42a6-9ed3-6eb199d94246/element/eyJmaW5kZXJUeXBlIjoiQnlUZXh0IiwidGV4dCI6IlRhcCBtZSEifQ==/text
[HTTP] {}
[debug] [W3C (3b2c33c6)] Calling AppiumDriver.getText() with args: ["eyJmaW5kZXJUeXBlIjoiQnlUZXh0IiwidGV4dCI6IlRhcCBtZSEifQ==","3b2c33c6-c325-42a6-9ed3-6eb199d94246"]
[debug] [FlutterDriver] Executing Flutter driver command 'getText'
[debug] [FlutterDriver] >>> {"command":"get_text","finderType":"ByText","text":"Tap me!"}
[FlutterDriver] {}
[debug] [W3C (3b2c33c6)] Encountered internal error running command: Error: {}
[debug] [W3C (3b2c33c6)]     at Object.wrappedLogger.errorAndThrow (/Users/kazuaki/.appium/appium-flutter-driver/node_modules/appium-flutter-driver/node_modules/@appium/support/lib/logging.js:94:35)
[debug] [W3C (3b2c33c6)]     at IsolateSocket.socket.call (/Users/kazuaki/.appium/appium-flutter-driver/node_modules/appium-flutter-driver/lib/sessions/observatory.ts:66:17)
[debug] [W3C (3b2c33c6)]     at runMicrotasks (<anonymous>)
[debug] [W3C (3b2c33c6)]     at processTicksAndRejections (internal/process/task_queues.js:95:5)
[HTTP] <-- GET /wd/hub/session/3b2c33c6-c325-42a6-9ed3-6eb199d94246/element/eyJmaW5kZXJUeXBlIjoiQnlUZXh0IiwidGV4dCI6IlRhcCBtZSEifQ==/text 500 5 ms - 559
```

Probably something was missed...

Reply:
still unknown the reason.
I've changed to start an app via intent method instead of monkey, but the same error. Tweaked `isolateId`, but the same error.

I only once succeeded to get an element after calling `activateApp` with current implementation, so I wonder something timing stuff exists to start an activity for android.